--- ./Makefile.in.org	Thu Nov 20 07:04:53 2003
+++ ./Makefile.in	Tue Dec 28 12:47:26 2004
@@ -35,17 +35,17 @@
 	${CC} ${CFLAGS} ${DEFS} -o $@ -c $<
 
 install: scponly debuglevel scponly.8
-	${INSTALL} -d ${bindir}
-	${INSTALL} -d ${mandir}/man8
-	${INSTALL} -d ${CONFDIR}
-	${INSTALL} -o 0 -g 0 scponly ${bindir}/scponly
-	${INSTALL} -o 0 -g 0 -m 0644 scponly.8 ${mandir}/man8/scponly.8
-	${INSTALL} -o 0 -g 0 -m 0644 debuglevel ${DEBUGFILE}
+	${INSTALL} -d ${DESTDIR}${bindir}
+	${INSTALL} -d ${DESTDIR}${mandir}/man8
+	${INSTALL} -d ${DESTDIR}${CONFDIR}
+	${INSTALL} scponly ${DESTDIR}${bindir}/scponly
+	${INSTALL} scponly.8 ${DESTDIR}${mandir}/man8/scponly.8
+	${INSTALL} debuglevel ${DESTDIR}${DEBUGFILE}
 	if test "x${CHROOTED_NAME}" != "x"; then			\
-		${INSTALL} -d ${sbindir};				\
-		rm -f ${sbindir}/${CHROOTED_NAME};			\
+		${INSTALL} -d ${DESTDIR}${sbindir};			\
+		rm -f ${DESTDIR}${sbindir}/${CHROOTED_NAME};		\
 		cp scponly ${CHROOTED_NAME};				\
-		${INSTALL} -o 0 -g 0 -m 4755 ${CHROOTED_NAME} ${sbindir}/${CHROOTED_NAME};	\
+		${INSTALL} ${CHROOTED_NAME} ${DESTDIR}${sbindir}/${CHROOTED_NAME};	\
 	fi
 
 debuglevel:
